Updating documentation of MAP_TOP_N_KEYS (#21239)
prestodb/presto: presto-docs/src/main/sphinx/functions/map.rst

.. function:: map_top_n_keys(x(K,V), n) -> array(K)

    Returns top n keys in the map ``x`` by sorting its keys in descending order.

>     ``n`` must be a non-negative integer

Reviewer: ```suggestion
    ``n`` must be a non-negative integer.
    
```
Suggest adding a period at the end of this sentence, and adding a line to put "For bottom ``n`` keys," at the left margin so it better introduces its code example.

    For bottom ``n`` keys, use the function with lambda operator to perform custom sorting

        SELECT map_top_n_keys(map(ARRAY['a', 'b', 'c'], ARRAY[3, 2, 1]), 2) --- ['c', 'b']

.. function:: map_top_n_keys(x(K,V), n, function(K,K,int)) -> array(K)

>     Returns top n keys in the map ``x`` by sorting its keys using the given comparator ``function``. The comparator will take

Reviewer: ```suggestion
    Returns top n keys in the map ``x`` by sorting its keys using the given comparator ``function``. The comparator takes
```

In general, avoid future tense such as "will" in documentation.

.. function:: map_top_n_keys(x(K,V), n) -> array(K)

>     Returns top n keys in the map ``x`` by sorting its keys in descending order.

Reviewer: ```suggestion
    Returns top ``n`` keys in the map ``x`` by sorting its keys in descending order.
```
Suggest for formatting  consistency with line 111.

    ``n`` must be a non-negative integer.

    For bottom ``n`` keys, use the function with lambda operator to perform custom sorting

        SELECT map_top_n_keys(map(ARRAY['a', 'b', 'c'], ARRAY[3, 2, 1]), 2) --- ['c', 'b']

.. function:: map_top_n_keys(x(K,V), n, function(K,K,int)) -> array(K)

>     Returns top n keys in the map ``x`` by sorting its keys using the given comparator ``function``. The comparator takes

Reviewer: ```suggestion
    Returns top ``n`` keys in the map ``x`` by sorting its keys using the given comparator ``function``. The comparator takes
```
Suggest for formatting  consistency with line 111.
